lib/screens/authentication/registration_screen.dart

```diff
@@ -124,3 +59,490 @@ class RegistrationBody extends StatelessWidget {
+              ),
+              const SizedBox(height: 16),
+              Container(
+                padding: const EdgeInsets.all(6),
+                child: TextField(
+                  decoration: InputDecoration(
+                    labelText: 'Confirm Password',
+                    prefixIcon: const Icon(Icons.vpn_key),
+                  ),
+                ),
+              ),
+              const SizedBox(height: 16),
+            ],
+          ),
+        );
+
+      case 2:
+        return Container(
+          padding: const EdgeInsets.symmetric(
+            horizontal: 8,
+            vertical: 16,
+          ),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              Text(
+                'General Information',
+                style: Theme.of(context).textTheme.subtitle1,
+              ),
+              const SizedBox(height: 16),
+              Container(
+                padding: const EdgeInsets.all(6),
+                child: DropdownButtonFormField(
+                  value: _currentDesignation ?? 'user',
+                  decoration: InputDecoration(
+                      labelText: 'Register',
+                      prefixIcon: const Icon(Icons.person_outline)),
+                  items: designations.map((designation) {
+                    return DropdownMenuItem(
+                      value: designation,
+                      child: Text('$designation Account'),
+                    );
+                  }).toList(),
+                  onChanged: (val) => setState(() => _currentDesignation = val),
+                ),
+              ),
+              const SizedBox(height: 16),
+              Expanded(
+                child: AnimatedSwitcher(
+                  child: _currentDesignation == 'user'
+                      ? Column(
+                          children: [
+                            Container(
+                              padding: const EdgeInsets.all(6),
+                              child: TextField(
+                                decoration: InputDecoration(
+                                  labelText: 'Age',
+                                  prefixIcon: const Icon(Icons.person),
+                                ),
+                              ),
+                            ),
+                            const SizedBox(height: 16),
+                            Container(
+                              padding: const EdgeInsets.all(6),
+                              child: TextField(
+                                decoration: InputDecoration(
+                                  labelText: 'sex',
+                                  prefixIcon: const Icon(Icons.email),
+                                ),
+                              ),
+                            ),
+                            const SizedBox(height: 16),
+                            Container(

```

Comment:
@wansom To save space, try and compact the first name and last name inside a row